### ISSUE TITLE
Is there a way to play multiple songs at once?

### ISSUE THREAD

**Issue by iam4159** [clementine-player/Clementine#6615]
I can't seem to find a setting for it

**Comment by iam4159** [comment]
Also, how do I run multiple instances at once?

**Comment by fmlatghor** [comment]
> Also, how do I run multiple instances at once?

Just run it twice? On kde you can do this:
![image](https://user-images.githubusercontent.com/52841429/77007120-08825400-693a-11ea-9fb0-6ffe1a75ee29.png)

If you run it from terminal, just open up another tab & run it again. Please don't be lazy, this isn't even an issue.

**Comment by jonaski** [comment]
That won't work.

**Comment by iam4159** [comment]
Yeah I found a solution which was to symlink /use/bin/clementine2 to
clementine then open then from terminal with nohup &

**Comment by fmlatghor** [comment]
Ha that's better than what I was trying to do; which was run a second instance inside a bwrap container.
-- 
 Securely sent with Tutanota. Get your own encrypted, ad-free mailbox: 
 https://tutanota.com


Mar 18, 2020, 20:09 by notifications@github.com:

>
>
>
> Yeah I found a solution which was to symlink /use/bin/clementine2 to
>  clementine then open then from terminal with nohup &
>
